service-api/src/main/java/greencity/dto/tariff/TariffsInfoDto.java

```diff
@@ -25,10 +25,8 @@ public class TariffsInfoDto {
     private List<ReceivingStationDto> receivingStations;
     private CourierDto courier;
     private String limitDescription;
-    private Long minAmountOfBags;
-    private Long maxAmountOfBags;
-    private Long minPriceOfOrder;
-    private Long maxPriceOfOrder;
+    private Long minQuantity;
+    private Long maxQuantity;
```

Comment:
`TariffsInfoDto` is used in `EmployeeDto` 
We return `EmployeeDto` to UI in the next endpoints: 

- `@PostMapping(value = "/save-employee")  `
- `@GetMapping("/getAll-employees")  `
- `@GetMapping("/getAll-active-employees") `
- `@PutMapping(value = "/update-employee"`

Have you check that all `UI` components wich use these endpoints work fine